ocw/lib/azure.py

```diff
@@ -85,13 +98,19 @@ def cleanup_all(self):
         storage_keys = [v.value for v in storage_keys.keys]
         block_blob_service = BlockBlobService(account_name='openqa', account_key=storage_keys[0])
 
+        keep_images = self.get_keeping_images()
+        self.cleanup_sle_images_container(block_blob_service, keep_images)
+        self.cleanup_disks_from_rg(keep_images)
+        self.cleanup_images_from_rg(keep_images)
+
+        for i in keep_images:
+            logger.info("[Azure] Keep image {} ".format(i))
```

Comment:
```suggestion
            logger.info("Keep image {} ".format(i))
```
Can we avoid this tag ? This is how log looks like now , and I clearly see from where message coming without it 
```
[36mpcw      |[0m 2020-03-10 17:27:31,553 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-..
[36mpcw      |[0m 2020-03-10 17:27:31,728 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:31,897 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:32,064 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:32,230 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:32,397 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:32,564 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-...
[36mpcw      |[0m 2020-03-10 17:27:32,731 ocw.lib.azure DEBUG    Found container bootdiagnostics-openqasus-..
[36mpcw      |[0m 2020-03-10 17:27:32,898 ocw.lib.azure INFO     [Azure] Delete container bootdiagnostics-...
```

Reply:
also logger has formatting within 
```suggestion
            logger.info("Keep image %d ",i)
```

Reply:
Ok.

I still think that the vault_namespace might help some times. So I would like to add a Log Adaper which will add this to all logmessages. But this is something for a different PR.

Reply:
vault_namespace would be [kernel-qa] , right ? If so I totally agree to add this and I already followed same approach here https://github.com/cfconrad/pcw/pull/77 . But [Azure] tag does not make much sense when you already have `ocw.lib.azure` just before it

Reply:
Ack. 
but lets not talk about it in this PR anymore :)